server/utils/applications/getResponses.ts

```diff
@@ -0,0 +1,21 @@
+import { FormArtifact, PageResponse } from '../../@types/ui'
+import { forPagesInTask } from './forPagesInTask'
+import { ApplicationOrAssessmentResponse, getSections } from './utils'
+
+export const getResponses = (formArtifact: FormArtifact): ApplicationOrAssessmentResponse => {
+  const responses = {}
+
+  const formSections = getSections(formArtifact)
+
+  formSections.forEach(section => {
+    section.tasks.forEach(task => {
+      const responsesForTask: Array<PageResponse> = []
+
+      forPagesInTask(formArtifact, task, page => responsesForTask.push(page.response()))
```

Comment:
This method gets introduced in a later commit. If it's not too much of a faff rebase wise, it'd be good to get the commits in the right order 🥺

Reply:
Was having such a mare with the history yesterday hence why it's a bit messy, I can sort it